TranslationApp/Program.cs

```diff
@@ -69,6 +69,7 @@ private static void UpdateAllTranslations()
         {
             using (new WaitCursorScope())
             {
+                GitModuleForm.IsUnitTestActive = true;
```

Comment:
A comment might be good here, as this is potentially confusing. Should the property be renamed?

Reply:
will do, thank you.